.github/workflows/main.yml

```diff
@@ -0,0 +1,70 @@
+name: Mac OS build
```

Comment:
@Krijger we stick to the Apple's current (2019) way of writing that: macOS

Reply:
check